### DISCUSSION TITLE
How would you quickly output text somewhere on screen that i can quickly plug variables to?

### DISCUSSION THREAD

**Discussion by somethingstarted** [musescore/MuseScore#16095] [Ideas]
[I was thinking something like the Unreal debugger text](https://www.google.com/search?q=unreal+engine+print+to+screen&rlz=1C1CHBF_enJP1023US1024&sxsrf=AJOqlzW0gdPfYnQ9SdglCtVCgBewW6pN3g:1674815208397&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjTg77sxOf8AhXlj2oFHQlXD_QQ_AUoAXoECAEQAw&biw=2048&bih=1063&dpr=1.25) it's very simple and you can quickly implement it, and multiple instances of it, and delete it when you dont want it anymore with ease. 

https://forums.unrealengine.com/t/print-to-screen-using-c/357351

do we have anything like that with musescore/qt? 

i am not sure what the best way to quickly put some text on screen that can easily be copy pasted into any section and then you plug in the things you need and it shows on the screen. At first i thought that i should do warning alerts, but if i'm looking at a number that could be changing hundreds of times, i might get hundreds of alert popups. thought about putting it in the bottom status bar, but that would interfere with, and be painful to setup, because you'd need to reference stuff in other areas that you may not have access to. thought about just creating a whole new window, but i wouldn't want to spend a whole month figuring out how to do that- just seems like a whole lotta nothing. 

(debugging, dev tools, development tools)

**Comment by cbjeukendrup** [comment]
Not sure I understand the question... could you elaborate a bit more? What kind of text do you mean, and what do you mean by pasting into any section, and what do you mean by plugging in the things you need?

**Reply by somethingstarted** [reply]
i updated my original message to elaborate some more.

**Comment by cbjeukendrup** [comment]
You should probably use console.log for this in QML or `LOGD() << ...` in C++. When it comes to C++, you can also use a real debugger.
